config.md

```diff
@@ -192,6 +192,18 @@ Index:
   files under the `MountPoint`. Users can turn it back on, by explicitly
   mentioning `Background: Build` in a later fragment.
 
+### StandardLibrary
+{:.v16}
```

Comment:
I think it's in v15 already.

(Can you check whether v15/v16 is already in the CSS?)